cfme/tests/control/test_actions.py

```diff
@@ -95,8 +58,8 @@ def get_vm_object(appliance, vm_name):
         return None
 
 
-@pytest.fixture(scope="module")
-def vm_name(provider):
+@pytest.fixture
+def vm_name():
     return random_vm_name("action", max_length=16)
 
```

Comment:
- As this fixture just returns random vm name string and you don't even need other fixture. We can define this as a global variable.

Reply:
Since this is a function scoped object, the vm name will be different for each test case. I think it'd be better to get rid of this altogether, since the VM name can be accessed through `vm.name` I will refactor the module to get rid of this.